include/fg/defines.h

```diff
@@ -141,23 +166,36 @@ enum dtype {
     f32 = 4,                                ///< Float (32-bits)
     s16 = 5,                                ///< Signed integer (16-bits)
     u16 = 6                                 ///< Unsigned integer (16-bits)
-};
-
-enum PlotType {
-    FG_LINE         = 0,
-    FG_SCATTER      = 1,
-    FG_SURFACE      = 2
-};
-
-enum MarkerType {
-    FG_NONE         = 0,
-    FG_POINT        = 1,
-    FG_CIRCLE       = 2,
-    FG_SQUARE       = 3,
-    FG_TRIANGLE     = 4,
-    FG_CROSS        = 5,
-    FG_PLUS         = 6,
-    FG_STAR         = 7
-};
-
+} fg_dtype;
```

Comment:
This will class with arrayfire, there is no reason to have them use the same enums.